edi-module/tests/test_utils.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/io;
+import ballerina/regex;
+import ballerina/file;
+
+function getSchema(string testName) returns EDISchema|error {
+    string schemaPath = check file:joinPath("tests", "resources", testName, "schema.json");
+    json schemaJson = check io:fileReadJson(schemaPath);
+    EDISchema schema = check schemaJson.cloneWithType(EDISchema);
+    return schema;
+}
+
+function getEDIMessage(string testName) returns string|error {
+    string inputPath = check file:joinPath("tests", "resources", testName, "message.edi");
+    return check io:fileReadString(inputPath);
```

Comment:
```suggestion
    return io:fileReadString(inputPath);
```

`check` is not required since it also does a return here. Just results in an additional check.